app/utils/nonsipp/check/LoansCheckStatusUtils.scala

```diff
@@ -46,11 +46,18 @@ object LoansCheckStatusUtils {
       case _ =>
         journeysStartedList
           .map(
+//            index => {
+//              refineV[OneTo5000](index.toInt + 1).fold(
+//                _ => List(false),
+//                refinedIndex => checkLoansRecord(userAnswers, srn, refinedIndex)
+//              )
+//            }
+
             index => {
-              refineV[OneTo5000](index.toInt + 1).fold(
-                _ => List.empty,
-                refinedIndex => checkLoansRecord(userAnswers, srn, refinedIndex)
-              )
+              refineV[OneTo5000](index.toInt + 1) match {
```

Comment:
Are you sure this is the same logic?

Reply:
see comment on BondsCheckStatusUtils.scala